src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Time.txt

```diff
@@ -67,6 +67,12 @@ false
 >> Time(1/0, 2, 3) < Time(8,1,30)
 #Error
 
+>> Date( 2019, 5, 16 ) > Time( 11, 23, 45 )
+true
+
+>> Time( 11, 23, 45 ) > Date( 2019, 5, 16 )
+false
+
```

Comment:
Can we have a comparison where the time is greater than the date? I believe something like `>> Date(1899, 12, 30) < Time(12,0,0)` would be true, since that date is the "date 0", and the time should be equivalent to 0.5